JavaScript/Endpoint Examples/JSON Payload/get-resource.js

```diff
@@ -1,10 +1,18 @@
 // This request demonstrates how to retrieve a resource using an ID.
 const axios = require('axios');
 
+// By default, we use the US-based API service. This is the primary endpoint for global use.
+var apiUrl = apiUrl + "";
```

Comment:
This is missing the US url

```suggestion
var apiUrl = "https://api.pdfrest.com";
```